docs/pipeline.rst

```diff
@@ -104,6 +107,29 @@ intent_featurizer_ngrams
           # feature vectors with character ngrams
           max_number_of_ngrams: 10
 
+intent_featurizer_count_vectors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:Short: Creates bag-of-words representation of intent features
+:Outputs: nothing, used as an input to intent classifiers that need bag-of-words representation of intent features (e.g. ``intent_classifier_tensorflow_embedding``)
+:Description:
+    Creates bag-of-words representation of intent features using sklearn's `CountVectorizer`
+
+:Configuration:
+
+    .. code-block:: yaml
+
+        pipeline:
+        - name: "intent_featurizer_count_vectors"
+          # regular expression for tokens
+          "token_pattern": r'(?u)\b\w\w+\b'
+          # min number of word occurancies in the document to add to vocabulary
+          "min_df": 1
+          # max number (fraction if float) of word occurancies in the document to add to vocabulary
+          "max_df": 1.0
```

Comment:
I think this comment could be clearer about what max_df is , maybe also link to sklearn docs